lib/steps.js

```diff
@@ -739,6 +739,20 @@ When(/I set the name of ?(?:the var(?:iable)?)? "([^"]+)" to "([^"]+)"/, async (
   ];
 
   let name_parts = name_str.split(' ');
+
+  if (name_parts.length > 4) {
+    let name_parts_last = name_parts[name_parts.length - 1];
+    let name_parts_slice = name_parts.slice(0, 3);
+    name_parts_slice.push(name_parts_last);
+    name_parts = name_parts_slice;
+  
+    await scope.addToReport(scope, {
+      type: `warning`,
+      value: `The name "${name_str}" has more than 4 parts, but 4 is maximum allowed. The test will set the name to "${name_parts.join(" ")}".`
```

Comment:
This is looking great! I'm not sure if we left "the" out of "is the maximum allowed" for v3, but I do think we should include "the" in v4. It's more grammatically correct, right? I think after that this will be ready!

Reply:
yikes- it think it is

Reply:
The part that needs updating now is the message itself to include "the". Once you've done that, the tests should pass.

Reply:
All tests passing!!!